lib/Runtime/Library/StringCacheList.h

```diff
@@ -8,6 +8,7 @@ STRING(Quotes, _u("\"\""))
 STRING(Whack, _u("/"))
 STRING(CommaDisplay, _u(","))
 STRING(CommaSpaceDisplay, _u(", "))
+STRING(Dot, _u("."))
```

Comment:
What uses this? I don't see any other references in this PR

Reply:
its used when building up the full display string as `library->GetDotString()`